src/app/devices/general/general.component.ts

```diff
@@ -101,17 +101,28 @@ export class GeneralComponent implements OnInit {
         })
       )
       .subscribe((results: any) => {
-        this.amtEnabledFeatures = this.fb.group({
-          enableIDER: results.amtFeatures.IDER,
-          enableKVM: results.amtFeatures.KVM,
-          enableSOL: results.amtFeatures.SOL,
-          userConsent: results.amtFeatures.userConsent,
-          optInState: results.amtFeatures.optInState,
-          redirection: results.amtFeatures.redirection
-        })
         this.generalSettings = results.generalSettings
         this.amtVersion = results.amtVersion
-        this.isDisabled = results.amtVersion?.AMT_SetupAndConfigurationService?.response?.ProvisioningMode === 4
+        this.isDisabled = results.amtVersion?.AMT_SetupAndConfigurationService?.response?.ProvisioningMode !== 1
+        if (!this.cloudMode) {
```

Comment:
i think can reduce the condition down to just setting disabled property for kvm ? kind of like line 106